deploy/helm/grafana-operator/values.yaml

```diff
@@ -55,6 +55,15 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+rbac:
+  # -- Specifies whether to create the ClusterRole and ClusterRoleBinding.
+  # If "namescapeScope" is true or "watchNamespaces" is set, this will create Role and RoleBinding instead.
+  create: true
+
+  # -- The rules affected to the ClusterRole.
+  # If not set and create is true, the rules are recovered from files/rbac.yaml or files/rbac-openshift.yaml.
+  rules: []
+
```

Comment:
Thanks for the PR!

First of all, I think the wording here is not perfect. - When I read the description, I'm not entirely sure what *"The rules affected to the `ClusterRole`"* means and why loading rules from `files/` folder might not be desirable.
The variable itself should have probably been named `ruleOverrides` to highlight that the values will replace whatever exists in the chart.

Second, in the PR description, you mentioned that not all of the requested permissions are needed for the operator to run. If that's the case, I think it should be resolved not through rule overrides (it would be a nightmare to maintain, so I doubt that many people would want to use the feature), but rather through modifying the code of the operator. - RBAC definitions are generated on the fly based on the comments in Go code, so it's just a matter of changing one line and then running a command to regenerate definitions. - Can't say for other maintainers, but, personally, I would argue that we should only accept the change for making RBAC templates optional (`rbac.create`), but not the rule overrides, so I would advise to remove the latter.

Reply:
Thank you for the feedback !

I agree with your suggestion to include only the `rbac.create` option and remove the `rbac.rules`, for the reasons you mentionned. I will update the PR.

Regarding the unnecessary permissions, my intention was to address specific use cases where not all features of the Grafana Operator would be required (`Ingress` creation/deletion for example). In this cases, it could be great to reduce the set of permissions.

I wasn't aware theses files were generated from the Go code and did wonder why they were there, thanks for pointing that out !

Reply:
Ah, alright, I thought it was a generic case. Excluding ingresses is a corner-case, so it can only be done using custom manifests.

I'll merge the PR, thanks for your contribution!
The chart will get released along with the next version of the operator (particular date is not decided yet, most likely, it's a matter of few weeks time).